src/services/user/useOnboardUser.ts

```diff
@@ -61,9 +65,9 @@ export function useOnboardUser(opts) {
     `
 
   return useMutation({
-    mutationFn: (input) => {
-      const formData = input?.formData
-      const selectedOrg = input?.selectedOrg
+    mutationFn: (input: { formData?: unknown; selectedOrg?: string }) => {
+      const formData = input.formData
+      const selectedOrg = input.selectedOrg
 
```

Comment:
I'll add zod schema validation in here - https://github.com/codecov/engineering-team/issues/2857